src/sentry/conf/server.py

```diff
@@ -2108,3 +2108,7 @@ def create_partitioned_queues(name):
 
 # Implemented in getsentry to run additional devserver workers.
 SENTRY_EXTRA_WORKERS = None
+
+DEMO_MODE = False
```

Comment:
This is probably worth a comment and warning what these enable by turning it on, since likely nobody would want to do this.

Reply:
@mattrobenolt good call, added a comment. Let me know if you think it needs to be changed.